background/get-addon-settings.js

```diff
@@ -194,6 +194,28 @@ chrome.storage.sync.get(["addonSettings", "addonsEnabled"], ({ addonSettings = {
             },
             manifest.presets.find((p) => p.id === "dark")
           );
+          updatePresetIfMatching(
+            settings,
+            3,
+            {
+              "motion-color": "#4C97FF",
```

Comment:
After some testing, migration might not go as expected if, for example, this setting is set to `#4c97ffff` instead of the default `#4C97FF`. These 2 represent the same color.

Reply:
Simply "looking" (clicking the setting without changing any sliders or clicking anything) into a color setting that allows transparency changes its value to lowercase and adds `ff` at the end to represent 100% alpha.

If we don't change this behavior, only users that applied the preset and never explored any color setting will be migrated.

Reply:
It's likely that we've missed this edge case in previous migrations, but it's too late to do anything about those now.

Reply:
See #5992